### ISSUE TITLE
Questions about the future of this repo

### ISSUE THREAD

**Issue by shalva97** [rohitneel1234/Android-Interview-Questions#2]
Hi. I have also created a small repo here https://github.com/shalva97/questions and decided to contribute, because you were the first to create the repo and most likely if we collaborate we can get more value out of it.

Here is my plan:
- remove strange tags in the README. instead of `<li>some questions here?</li>` there will be `- some questions here?`.
- send PRs for all other questions I have in my repo
- split questions into more different categories

Regarding answers, I dont really want to type that much, meaning PRs from me will be just list of questions or cleanups. In rare cases I will also add links to a possible explanation(usually links break, so I will try to only link from popular websites like Medium or StackOverflow).

What do you think?

**Comment by rohitneel1234** [comment]
Yes I am fine with your approach. We can collaborate and contribute each other in repo. I will start sending PR in your repo soon.
